src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -140,6 +140,8 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             ApplySettings(settings, Settings);
```

Comment:
This method makes async calls but no cancellation token is provided anywhere. Consider adding a cancellation token. If it's too much work, we can follow up on that.